project.clj

```diff
@@ -10,7 +10,7 @@
                  [re-frame "1.2.0"]
                  ;; handlerForForeign
                  [com.cognitect/transit-cljs "0.8.256"]
-                 [re-frisk/sente "1.20.0"]
+                 [com.taoensso/sente "1.20.0"]
```

Comment:
could you please elaborate on why this change is needed? i believe there was a reason why i forked it, i just don't remember :)

Reply:
ok i found, we just need to use latest version

Reply:
so that's fine